app/views/events/_upcoming_events.html.erb

```diff
@@ -0,0 +1,31 @@
+<section>
```

Comment:
Note: this will render the rest of my comments moot, but I've left them in for reference (but yes, I think you can close them, this is the most important comment)